modules/003-deckhouse-config/crds/module-config.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: moduleconfigs.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: deckhouse-config
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: moduleconfigs
+    singular: moduleconfig
+    kind: ModuleConfig
+  preserveUnknownFields: false
+  versions:
+    - name: v1
```

Comment:
Do we have to start with version `v1`? Not `v1alpha1`? Maybe we will find some desired changes here...